File: cpp/trig.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
  string x;
  cout << "sin/cos/tan>";
  cin >> x;
  if (!x.compare("sin")) {
    int sine;
    cout << "Type the number you want to sine>";
    cin >> sine; 
    cout << sin(sine);
  } else if (!x.compare("cos")){
    int cosine;
    cout << "Type the number you want to cosine>";
    cin >> cosine;  
    cout << cos(cosine);
  } else if (!x.compare("tan")){
    int tangent;
    cout << "Type the number you want to tangent>";
    cin >> tangent; 
    cout << tan(tangent);
  }
  return 0;
}

File: python/trig.py
import math
import time


choose = input('sin/cos/tan>')

if choose == 'sin':
    sine = int(input('Type the number you want to sine>'))
    print('The answer is', math.sin(sine))

elif choose == 'cos':
    cosine = int(input('Type the number you want to cosine>'))
    print('The answer is', math.cos(cosine))

elif choose == 'tan':
    tangent = int(input('Type the number you want to tangent>'))
    print('The answer is', math.tan(tangent))

print('A countdown has been set because the program will close after the code is done.')


def countdown(time_sec):
    while time_sec:
        mins, secs = divmod(time_sec, 60)
        timeformat = '{:02d}:{:02d}'.format(mins, secs)
        print(timeformat, end='\r')
        time.sleep(1)
        time_sec -= 1

    print("Bye!")


countdown(10)
